main.js

```diff
@@ -1 +1,30 @@
 'use strict';
+function createCalendar(id, year, month) {
+  var elem = document.getElementById(id);
+    let mon = month - 1;     
```

Comment:
It is not good to use short names like this, will be better to name `monthFromZero` or `decrementedMonth` etc.

Reply:
prefer const, use let only if you need to change the variable in future